vscode/core/CHANGELOG.md

```
@@ -1,9 +1,162 @@
+- Wait for extension to initialize
+
+### New Contributors
+
+- [@rhuanhianc](https://github.com/rhuanhianc)
+- [@jmontleon](https://github.com/jmontleon)
+- [@abrugaro](https://github.com/abrugaro)
+- [@feiskyer](https://github.com/feiskyer)
+- [@RanWurmbrand](https://github.com/RanWurmbrand)
+- [@fabianvf](https://github.com/fabianvf)
+
+**Full Changelog**: https://github.com/konveyor/editor-extensions/compare/v0.1.0...v0.2.0
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)